### ISSUE TITLE
Make it easier to find information on sharing essays

### ISSUE THREAD

**Issue by densnow** [dsa-ou/algoesup#7]
The documentation on how to share finished essays is not that easy to find on the GitHub pages site. Most of the information is buried away in the Deepnote how to guides, and is linked to via other sections, such as the writing and feedback guides. 

My initial thoughts were: we have a "quick start" guide, but we don't have a "quick finish" guide. Obviously the name is not suitable, but the sentiment is what I am thinking about.

Some suggestions:
- Some kind of "quick guide" as to what one should do when ones essay is finished; written in the same spirit as the quick start guide.
- Add specific links and a quick paragraph (that is to be deleted) to the end of the template essays to remind authors on what to do when finished and how to share their essays.
- All of the above.

**Comment by mwermelinger** [comment]
Good points. Maybe both templates need to have links throughout them to the relevant guide sections, from how to choose a problem to how to share for commenting. While we can link directly to permalinks in the guide, maybe we should break the writing and feedback guide into separate section pages.

**Comment by densnow** [comment]
Yes, I think additional links in the templates could work well. 

Maybe we need a "Sharing your essays"  or similarly named section that links to the feedback guide and also contains the technical details of how and where to share essays. We could move some parts from the how-to guides and replace them with links to the new location to keep it DRY

I am hoping this issue can be guided in part by feedback received from users of the resources.

**Comment by densnow** [comment]
It seems we both agree that the templates need some additional content in the form of links and hints. I will open a new issue to address this specific concern.

**Comment by mwermelinger** [comment]
By making all getting started guides having the same structure, including how to share, this should be covered.
